### PR TITLE
Add support for night according to Rabenou Tam and mincha gedola according to 30 min

### PR DESCRIPTION
**Pull request by DanBendavid**
# Description
Adding Night by Rabenou Time in the Zmanim
	
modified:   hdate/zmanim.py
modified:   hdate/htables.py
modified:   tests/test_hdate_api.py
        

# Closes issue(s)

  Fixes: #<issue number>

# Changes included (select only one)

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible)

# Checklist

- [ ] Code passes tox

### CODE REVIEW

**Comment by tsvi** [comment]
Much better.
Please do not change mincha gedola, rather add the other shita.
The current calculation is in accordance with mishna brura in שער הציון רל"ג ח'
image

But we should support other calculations as well. Let's call it big_mincha_30_min

**Comment by tsvi** [comment]
Also you're failing black formatting:
`pdm run black` (Should not happen if you have pre-commit installed)

File: README.rst [hunk 1]
```diff
@@ -38,10 +38,12 @@ base code to provide times of the day in hebrew:
     סוף זמן תפילה גר"א - 10:28:00
     חצות היום - 12:40:00
     מנחה גדולה - 13:10:30
+    מנחה גדולה ל - 13:10:00
```
Comment (tsvi) [inline README.rst:41]: Much better, please change in Hebrew to 
מנחה גדולה 30 דק

The current wording is not clear.
After that I will merge this PR.

**Review by tsvi — CHANGES_REQUESTED** [review]

**Comment by DanBendavid** [comment]
I will work on multilanguage this sunday. . 
What do you recommend ? Cause moving from hebrew=true to lang=hebrew is going to be a breaking change.

**Review by tsvi — APPROVED** [review]
Looks good.
Thank you and kol hakavod.

**Comment by tsvi** [comment]
> I will work on multilanguage this sunday. . What do you recommend ? Cause moving from hebrew=true to lang=hebrew is going to be a breaking change.

It's ok to have a breaking change. I think it's a good idea. Let's change it from hebrew to language.
The internal implementation we can change later.

**Pull request merged**